### ISSUE TITLE
folder search?

### ISSUE THREAD

**Issue by ehahn9** [azai91/alfred-drive-workflow#62]
Hi Alexander! I use your wonderful gdrive alfred workflow daily! Would you be willing to add support to finding gdrive folders as well as files? Not sure if that's even doable? Basically "d foo" would match foo/ and open the google folder/tag. This sort of parallels alfred's simple folder search workflow ("f foo").

If a small donation$ would help, I'd be honored!

**Comment by cmeinerd** [comment]
+1 I also would love to see this feature

**Comment by cmeinerd** [comment]
Hi @azai91,

any update if this is possible or on the roadmap would be great.

I really like your workflow and using it everyday.
I just miss the functionality to open folders from Google Drive in finder.

Thanks,
Christian

**Comment by GabLeRoux** [comment]
yeah that would be great indeed. I feel this is the only thing missing here 👍 	Good job with the plugin, it works very well and it's very useful.

**Comment by johnathanz** [comment]
You can folder search with a separate Alfred plugin: https://github.com/login?return_to=%2Fazai91%2Falfred-drive-workflow

**Comment by cmeinerd** [comment]
@johnathanz seems the link redirects to this plugin, can you share the correct link?

Thanks,
Christian

**Comment by GabLeRoux** [comment]
@johnathanz which plugin are you talking about?

**Comment by aMittalAnshul** [comment]
I'd love this feature to show folders as well.  Isn't there a way we can configure this ourselves?

**Comment by dabeeeenster** [comment]
Ive tweaked the code to do this and forked the project here:

https://github.com/dabeeeenster/alfred-drive-workflow

**Comment by cmeinerd** [comment]
@dabeeeenster How can I try it out?

**Comment by dabeeeenster** [comment]
Ive made a release here https://github.com/dabeeeenster/alfred-drive-workflow/releases/tag/v2.2

**Comment by GabLeRoux** [comment]
I confirm it works. The diff is here: https://github.com/dabeeeenster/alfred-drive-workflow/commit/c2526817f045387d1e373179018d7ce72e61d2f9

I would remove comments and would instead add a setting to enable/disable folder search. I'd let plugin author (@azai91) decide wether this should be enabled or disabled by default. But I'd personally enable folders by default.

Here's how users can set a setting with alfred workflows:

![image](https://user-images.githubusercontent.com/1264761/60035779-a3842100-967b-11e9-8e86-a4efc612f246.png)

![image](https://user-images.githubusercontent.com/1264761/60035822-bc8cd200-967b-11e9-91f8-2e0b412a55c2.png)

If the setting was added with the `Don't Export` flag, it will already be there and part of the exported workflow.
